VTUL/dlp-access
# Add typescript dependencies and config

**Add typescript dependencies and config to support using typescript.**
* * *

**JIRA Ticket**: (link) (:star:)

* Other Relevant Links (Meeting note, project page, related pull requests, etc.)

# What does this Pull Request do? (:star:)
Add typescript dependencies and config to support using typescript.

# What's the changes? (:star:)
* Add typescript dependencies and config to support using typescript.

# How should this be tested?

A description of what steps someone could take to:
* delete `node_modules` directory and then run `npm install`
* Create a simple functional component using typescript syntax and render it.
* Check that new component does not cause any errors

# Additional Notes:
Any additional information that you think would be helpful when reviewing this PR.
* branch: `typescript` 

# Interested parties
@andreaWaldren 

(:star:) Required fields
